NewsBook/Application/Commands/FavouriteNews/FavouriteNewsCommandsHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using AutoMapper;
+using MediatR;
+using NewsBook.Mediator.Commands.FavouriteNewsCommands;
+using NewsBook.Mediator.Response;
+using NewsBook.Repository;
+using NuGet.Protocol.Plugins;
+
+namespace NewsBook.Mediator.Commands.FavouriteNewsCommand
+{
+    public class FavouriteNewsCommandsHandler : 
+        IRequestHandler<MarkFavouriteNews, NewsReadDTO>,
+        IRequestHandler<DeleteFavouriteNews, NewsReadDTO>
+    {
+        private readonly IFavouriteNewsRespository _favouriteNews;
+        private IMapper _mapper;
+        public FavouriteNewsCommandsHandler(IFavouriteNewsRespository favouriteNews, IMapper mapper)
+        {
+            _favouriteNews = favouriteNews;
+            _mapper = mapper;
+        }
+
+        public async Task<NewsReadDTO> Handle(MarkFavouriteNews request, CancellationToken cancellationToken)
+        {
+            var favouriteNews = await _favouriteNews.Insert(
```

Comment:
Don't use unnecessary line breaks. Split code to multiple lines only if required and makes the code more readable.